architecture/core_concepts/routes.adoc

```diff
@@ -229,6 +229,7 @@ $ oc set env dc/router HAPROXY_ROUTER_SYSLOG_ADDRESS=127.0.0.1 HAPROXY_ROUTER_LO
 |`*ROUTER_SERVICE_NAMESPACE*` |  | The namespace the router identifies itself in the in route status. Required if `ROUTER_SERVICE_NAME` is used.
 |`*ROUTER_SERVICE_NO_SNI_PORT*` | 10443 | Internal port for some front-end to back-end communication (see note below).
 |`*ROUTER_SERVICE_SNI_PORT*` | 10444 | Internal port for some front-end to back-end communication (see note below).
+|`*ROUTER_SLOWLORIS_HTTP_KEEPALIVE*` | 300s | Set the maximum time to wait for a new HTTP request to appear.
```

Comment:
ROUTER_SLOWLORIS_TIMEOUT is a request must complete in this time. ROUTER_SLOWLORIS_HTTP_KEEPALIVE is needed because when "timeout http-request" is set and "timeout http-keep-alive" is not haproxy defaults the ladder to the former and a small "timeout http-keep-alive" can confuse browsers/apps that are not expecting such a small keep alive